index.html

```diff
@@ -6,213 +6,237 @@
-        </li>
-        <li>
-            <img src="hero/Linkedin icon.svg" alt="">
-        </li>
-        <li><img src="hero/3.svg" alt="">
-        </li>
-        <li>
-            <img src="hero/4.svg" alt="">
-        </li>
-        <li>
-            <img src="hero/5.svg" alt="">
-        </li>
-    </ul>
-    <p class="button">Get my resume</p>
-    <ul>
-        <li class="lang">Languages <img src="hero/left chi.png" height="25px" alt="">
-                <li class="languages"><img src="hero/js.png" width="50px" height="50px" alt=""><p>JavaScript</p> </li>
-
-                <li class="languages">
-                    <img src="hero/html.png"width="50px" height="50px" alt=""><p>HTML</p>
+<div class="LION">
+<section class="myself">
+    <div class="for-flex">
+        <div class="ttwo"><br>
+        <div class="D">
+            <h1 class="about">About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+             <p class="connect">LET’S CONNECT</p>
+             <ul class="social-links">
+                 <li>
+                <img src="hero/1.svg" alt="">
                 </li>
+            <li>
+               <img src="hero/Linkedin icon.svg" alt="">
+            </li>
+            <li>
+                <img src="hero/3.svg" alt="">
+            </li>
+            <li>
+                 <img src="hero/4.svg" alt="">
+            </li>
+            <li>
+                <img src="hero/5.svg" alt="">
+            </li>   
+            </ul>
+            <p class="button">Get my resume</p>
+        </div> 
+    </div>
+    <div class="LLI">
+         <ul id="LI">
+            <div class="parent language">
+                <p class="Tt lang">Language <img id="chirvon" src="hero/left chi.png"  height="20px" width="20px" alt=""></p>
+                <div class="child"><img id="S" src="hero/js.png" alt=""><div class="T">Javascript</div></div>
         
-                <li class="languages">
-                    <img src="hero/css.png" width="50px" height="50px" alt=""><p>CSS</p>
-                </li>
-        </li>        
-        <li class="lang">
-                    Frameworks <img src="hero/chirvon.png" height="25px"  alt="">
-        </li>
-        <li class="lang">
-                    Skills  <img src="hero/chirvon.png"  height="25px" alt="">
-        </li>
+                <div class="child"><img id="S" src="hero/html.png" alt=""><div class="T">HTML</div></div>
         
+                <div class="child"><img id="S" src="hero/css.png" alt=""><div class="T">CSS</div></div>
+            </div>
+        
+            <div class="lang">
+                <p class="Tt">Framework </p>
+                <img id="chirvon" src="hero/chirvon.png" height="20px" width="20px" alt="">
+            </div>
+            <hr class="line">
```

Comment:
@Sohail-Aslam Your size/spacing is not matching to the original design please consider to make it similar 🆒 😎 
![sohail-2](https://github.com/Sohail-Aslam/Mobile-Version/assets/122353030/47037fa8-f2f9-4d0e-9dd8-48a62dc66f43)